curvefs/src/tools/BUILD

```diff
@@ -14,10 +14,71 @@
 #  limitations under the License.
 #
 
-load(
-    "//:copts.bzl",
-    "GCC_FLAGS",
-    "GCC_TEST_FLAGS"
+COPTS = [
+    "-DGFLAGS=gflags",
+    "-DOS_LINUX",
+    "-DSNAPPY",
+    "-DHAVE_SSE42",
+    "-DNDEBUG",
+    "-fno-omit-frame-pointer",
+    "-momit-leaf-frame-pointer",
+    "-msse4.2",
+    "-pthread",
+    "-Wsign-compare",
+    "-Wno-unused-parameter",
+    "-Wno-unused-variable",
+    "-Woverloaded-virtual",
+    "-Wnon-virtual-dtor",
+    "-Wno-missing-field-initializers",
+    "-std=c++11",
+    "-Wall",
+    "-Werror",
+]
+
+cc_binary(
+    name = "curvefs_tool",
+    srcs = glob(["curvefs_tool_main.cpp"]),
+    copts = COPTS,
+    linkopts = ["-L/usr/local/lib/x86_64-linux-gnu"],
```

Comment:
this is also unneeded, you should fix this later

Reply:
> this is also unneeded, you should fix this later

fix